content/courses/ada_2022_whats_new/chapters/enum_val.rst

```diff
@@ -0,0 +1,179 @@
+.. include:: ../../global.txt
+
+Enumeration representation
+==========================
+
+.. note::
+
+    Enumeration representation attributes are supported by
+
+    * GNAT Community Edition 2019
+    * GCC 9
+
+Enumeration types in Ada are represented as integers on the machine
+level. But, actually, there are two mappings from enumeration to
+integer: a literal position and a representation value.
+
+Literal positions
+-----------------
+
+Each enumeration literal has corresponding position in the type
+declaration. We can get it easily through :ada:`Type'Pos (Enum)`
+attribute.
+
+.. code-block:: ada
```

Comment:
Instead of having static code blocks and the output below, we should really make use of the widget here.

Reply:
(This applies to the other code blocks below...)